include/mgos_pcf857x.h

```diff
@@ -63,6 +66,14 @@ void mgos_pcf857x_gpio_clear_int(struct mgos_pcf857x *dev, int pin);
 void mgos_pcf857x_gpio_remove_int_handler(struct mgos_pcf857x *dev, int pin, mgos_gpio_int_handler_f *old_cb, void **old_arg);
 bool mgos_pcf857x_gpio_set_button_handler(struct mgos_pcf857x *dev, int pin, enum mgos_gpio_pull_type pull_type, enum mgos_gpio_int_mode int_mode, int debounce_ms, mgos_gpio_int_handler_f cb, void *arg);
 
+/*
+ * A utility function that takes care of blinking an LED.
+ * The pin must be configured as output first.
```

Comment:
Why would you not set the pin to output as a part of the implementation. Calling blink() on an input pin seems silly.

Reply:
You0re right. I configure the mode as output when blink is called.